src/main/java/meteordevelopment/meteorclient/mixin/BeaconScreenMixin.java

```diff
@@ -30,7 +30,7 @@
 @Mixin(BeaconScreen.class)
 public abstract class BeaconScreenMixin extends HandledScreen<BeaconScreenHandler> {
     @Shadow
-    protected abstract <T extends ClickableWidget> void addButton(T button);
+    protected abstract <T extends ClickableWidget & BeaconScreen.BeaconButtonWidget> void addButton(T button);
```

Comment:
This change was previously undone because it causes a crash. Despite it fixing a mixin "issue", it makes it unable to find the target method.